argocd/overlays/moc-infra/projects/kruize.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
The resource manifest looks good to me. In order to get it deployed please list it in the [`kustomization.yaml`](https://github.com/operate-first/apps/blob/master/argocd/overlays/moc-infra/projects/kustomization.yaml)